html/bible_browser_toolbox.html

```diff
@@ -41,9 +41,7 @@ <h3 class="ui-accordion-header ui-helper-reset ui-state-default ui-state-active
 </div>
 
 <div id="dictionary-info-box" class="accordion-panel">
-  <h3>
-    <a href="#" id="dictionary-info-box-header" i18n="[html]dictionary-info-box.default-header"></a>
-  </h3>
```

Comment:
Not sure why these changes were introduced, please restore.

Reply:
very strange i did not touched that one :( Maybe at rebase. 
Let me  close this one and do it again as i have the changes locally